.github/workflows/linters.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  workflow_call:
+
+env:
+  NODE_ENV: ci
+
+name: "Linters"
+
+jobs:
+  lint:
+    name: Code Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - run: npm ci --ignore-scripts
+      - run: npm run lint
+
+  commit-lint:
+    name: Commit Lint
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - run: npm ci --ignore-scripts
+      - name: Validate PR commits with commitlint
+        run: npx commitlint --from ${{ github.event.pull_request.base.sha }} --to ${{ github.event.pull_request.head.sha }} --verbose
```

Comment:
Since we squash most of our PRs, should we commit lint the title of the PR instead of the individual commits, which will be squashed anyway? The title of the PR will be the committed name that will eventually be on the actual history.😅

I think we can keep what you have. If we want to lint the PR title, we can make use of `github.event.pull_request.title` and pass it to commitlint.

Reply:
Oops... I just realise this PR has been merged. It's ok, just a small thing. haha